front/app/services/users.ts

```diff
@@ -25,7 +25,7 @@ export interface IUserAttributes {
   birthyear?: number;
   domicile?: string;
   education?: string;
-  unread_notifications?: number;
+  unread_notifications: number;
```

Comment:
@jinjagit double checking: this user attribute can never be undefined, right?

New account had 0. Existing account with notifications had 0 as well after reading notifications.

Reply:
Looks like it can't be undefined, no. A user object collects `unread_notifications` according to...
```
has_many :unread_notifications, -> { where read_at: nil }, class_name: 'Notification', foreign_key: :recipient_id
```
So, I assume we get the number (probably on the FE) by counting the number of objects in the returned collection, which would be 0 if the collection is empty (or nothing is found).

Reply:
On the BE, this seems to work OK.

Testing locally:
1. Create a new user, check `user.unread_notifications.count` = 0
2. New user creates an idea + Admin gives official feedback on idea
3. `user.unread_notifications.count` = 1
4. New user read notification
5. `user.unread_notifications.count` = 0

Reply:
Thx 👍